### PR TITLE
wrong parameter usage. 

### PR DESCRIPTION
**Pull request by cc5092**
spanId will set by parentId and it will be null when it is root span

### CODE REVIEW

File: src/Zipkin/TraceContext.php [hunk 1]
```diff
@@ -162,13 +176,34 @@ public function withSampled($isSampled)
...
             $this->isDebug
         );
     }
 
+    /**
+     * check traceId128bits flag
+     * change trace id to 128bits if necessary
+     * @param bool $traceId128bits
+     */
+    public function checkTraceId128bits($traceId128bits = false)
```
Comment (jcchavezs) [inline src/Zipkin/TraceContext.php]: This method worries me for two reasons: 
1. Makes the TraceContext mutable (which I'd discourage)
2. It receives one parameter but only does something if the parameter is `true`, I'd go for a semantic method like `useTraceId128bits()` but again I'd avoid mutation methods.

What do you think?

File: src/Zipkin/TraceContext.php [hunk 2]
```diff
@@ -33,13 +33,19 @@
      */
     private $parentId;
 
-    private function __construct($traceId, $spanId, $parentId, $isSampled, $debug)
+    /**
+     * @var bool
+     */
+    private $traceId128bits;
```
Comment (jcchavezs) [inline src/Zipkin/TraceContext.php]: What do you think about calling this `useTraceId128bits`? IMO that is more descriptive and works better as flag.

File: src/Zipkin/TraceContext.php (hunk 1 above)
Comment (cc5092) [reply src/Zipkin/TraceContext.php]: $tracer->newTrace() function calls TraceContext::createAsRoot() to create a root span context. 

TraceContext::createAsRoot() will create a 64bits id for trace id and span id.

createAsRoot() has no parameters to know if 128bits trace id is needed

Comment (jcchavezs) [reply src/Zipkin/TraceContext.php]: Yeah, that is true, we could have it like `createAsRoot(SamplingFlags $samplingFlags = null, $useTraceId128bits = false)` and that parameter coming from the tracer. What do you think?

Comment (cc5092) [reply src/Zipkin/TraceContext.php]: I think it's ok. I will update my pr.

Comment (cc5092) [reply src/Zipkin/TraceContext.php]: @jcchavezs updated~

**Comment by jcchavezs** [comment]
Awesome work @cc5092! 🥇

**Pull request merged**